react/features/subtitles/reducer.js

```diff
@@ -1,15 +1,16 @@
 import { ReducerRegistry } from '../base/redux';
 
 import {
-    REMOVE_TRANSCRIPT_MESSAGE,
+    REMOVE_TRANSCRIPT_MESSAGE, TOGGLE_REQUESTING_SUBTITLES,
     UPDATE_TRANSCRIPT_MESSAGE
 } from './actionTypes';
 
 /**
  * Default State for 'features/transcription' feature
  */
 const defaultState = {
-    transcriptMessages: new Map()
+    _transcriptMessages: new Map(),
+    _requestingSubtitles: false
```

Comment:
Why the underscores?

Reply:
I added underscored because I thought these properties should be private through the subtitles feature.